Deep_Learning/Dog Breeds Classification/web-app/src/App.css

```
@@ -0,0 +1,129 @@
+  max-width: 50vw;
+  height: 40vh;
+  border-radius: 10px;
+  margin-top: 20px;
+}
+
+.onupload h1 {
+  font-size: 30px;
+  /* background-color: rgb(133, 245, 245); */
+  color: #000000;
+  margin-top: 10px;
+  margin: auto;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)